Fakebook.Profile/Fakebook.Profile.RestApi/Program.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Fakebook.ProfileRestApi
```

Comment:
the namespace here & in startup doesnt match the folder heirarchy, which we usually expect